src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R4/DocumentReference-example-relatesTo-code-appends.indexes.json

```diff
@@ -296,4 +296,4 @@
       "isValidAsCompositeComponent": true
     }
   }
-]
```

Comment:
does changes in this files necessary

Reply:
These files were originally updated as part of this feature branch but then got merged earlier into main. Removed the extra whitespace